pkg/config/config_template.yaml

```diff
@@ -1272,6 +1272,12 @@ api_key:
   #   - 'sql*'
   #   - '*pass*d*'
 
+  ## @param disable_realtime - boolean - optional - default: false
+  ## @env DD_PROCESS_CONFIG_DISABLE_REALTIME - boolean - optional - default: false
+  ## Disable realtime process and container checks
+  #
+  # disable_realtime: false
```

Comment:
Minor nit and not a blocker but probably `disable_realtime_checks` would be a better fit for the config field name here

Reply:
Good, point, thank you @sgnn7 , will update